tests/simpleKVBC/TesterReplica/setup.cpp

```diff
@@ -243,4 +270,50 @@ std::unique_ptr<IStorageFactory> TestSetup::GetStorageFactory() {
   return std::make_unique<v2MerkleTree::RocksDBStorageFactory>(dbPath.str());
 }
 
+std::vector<std::string> getKeyDirectories(const std::string& keysRootPath)
```

Comment:
nit: You don't check the strict folder names 1,2,...
This is fine for Apollo but not for production :)